src/mappings/icd11foundation.sssom.tsv

```diff
@@ -0,0 +1,11 @@
+# curie_map:
+#   owl: http://www.w3.org/2002/07/owl#
+#   rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+#   rdfs: http://www.w3.org/2000/01/rdf-schema#
+#   semapv: https://w3id.org/semapv/vocab/
+#   skos: http://www.w3.org/2004/02/skos/core#
+#   sssom: https://w3id.org/sssom/
+# license: https://w3id.org/sssom/license/unspecified
+# mapping_set_id: https://w3id.org/sssom/mappings/8b5fc9cd-51d1-4baa-a4de-80c9dd83e0f2
+
```

Comment:
## Some empty mapping files
- [ ] @matentzn Double check

I believe you reviewed this already in a previous build of the ICD11 PR before we merged it, and I think this is OK, but we never discussed it, so I just wanted to double check.

OK files (>1k lines):
- src/ontology/lexmatch/unmapped_icd11foundation_lex.tsv 
- src/ontology/lexmatch/unmapped_icd11foundation_lex_exact.tsv

Empty files:
- src/mappings/icd11foundation.sssom.tsv
- src/ontology/lexmatch/mondo-only/unmapped_icd11foundation_mondo.tsv
- src/ontology/lexmatch/mondo-only/unmapped_icd11foundation_mondo_exact.tsv